bitbots_body_behavior/src/bitbots_body_behavior/actions/kick_ball.py

```diff
@@ -11,6 +11,26 @@ def perform(self, reevaluate=False):
             self.blackboard.animation.play_animation(self.right_kick)
 
 
+class KickBallRight(AbstractActionElement):
+    def __init__(self, blackboard, dsd, parameters=None):
+        super(KickBallRight, self).__init__(blackboard, dsd, parameters)
+        self.right_kick = 'kick_right'  # TODO get actual name of parameter from some config
+
+    def perform(self, reevaluate=False):
+        if not self.blackboard.animation.is_animation_busy():
+            self.blackboard.animation.play_animation(self.right_kick)
+
+
+class KickBallLeft(AbstractActionElement):
+    def __init__(self, blackboard, dsd, parameters=None):
+        super(KickBallLeft, self).__init__(blackboard, dsd, parameters)
+        self.right_kick = 'kick_left'  # TODO get actual name of parameter from some config
```

Comment:
`right_kick` is probably not the best name here. I guess `KickBallLeft` and `KickBallRight` could be merged and a DSD parameter could be used to distinguish the direction.

Reply:
Oh. I just didn't see that. Using the parameter might be the best option.